content/en/docs/setup/install/_index.md

```diff
@@ -6,35 +6,69 @@ description: >
 
-* A Kubernetes cluster on which to install Spinnaker itself
+* A [Kubernetes cluster](/docs/setup/install/providers/kubernetes-v2/) on which to install Spinnaker
+  * It's recommended to have at least 8 cores and 32GB of RAM available in the cluster. 
+  * It's recommended to use SSDs or high IO disks (GP3) due to spinanker using kubectl for operations
+  * You can deploy to k3s/microk8s or similar for on-prem but keep in mind avaialble capacity
 
-  We recommend at least 4 cores and 16GB of RAM available in the cluster. 
+* Supoprting infrastructure not all of which is documented here, but should be considered
+  * DNS
+  * Load balancers (ALB/Traefik/etc.)
+  * SSL and certificates
+  * Backups
+  * ETC
 
-You can also install [on a single local machine](https://www.spinnaker.io/setup/install/environment/#local-debian), or [for Spinnaker development](https://www.spinnaker.io/setup/install/environment/#local-git), making sure you have the 4 cores and 16GB in each case. 
+>  Other installation targets may work but are not supported.  For large scale production Kubernetes is recommended:
+>  * [Local development](https://www.spinnaker.io/setup/install/environment/#local-git)
+>  * [Debian local](https://www.spinnaker.io/setup/install/environment/#local-debian) 
+>  * [Vagrant example (uses local debian)](hhttps://github.com/ashleykleynhans/vagrant-jenkins-spinnaker/)
 
-## The process
+## Installation tools
+There are several solutions around deploying spinnaker to kubernetes.  Please note that not all these solutions are officially supported at this time though several are well maintained by the community.
+- [halyard](/docs/setup/install/halyard) Is a command line tool that has a CLI and API component.  Halyard creates a small, 
+ headless Spinnaker to update Spinnaker and its microservices, ensuring zero-downtime updates.  This is the default 
+ installation method.  This takes care of creating Kubernetes objects from halyard configuration files.  The community
+ would like to replace this tool, but it's still the default installation method that works for most users.
+- [Armory Spinnaker Operator for Kubernetes](https://github.com/armory/spinnaker-operator) is an open source Kubernetes
+Operator for deploying and managing Spinnaker. You can install a basic version of Spinnaker or use Kustomize files for
+advanced configuration.  There's a [configuration reference](https://docs.armory.io/continuous-deployment/installation/armory-operator/op-manifest-reference/)
+that documents a large part of the available configuration of spinnaker. 
+- [Kustomize native install](https://github.com/spinnaker/spinnaker-kustomize/) is a minimal install that provides
+a native kubernetes deployment experience for the spinnaker microservices.  This provides a more native k8s deployment
+without the need for helm or an operator or halyard CLI tooling.  Caution:  This uses things LIKE a mariadb driver which
+is not currently available in the project by default.  
```

Comment:
```suggestion
without the need for helm or an operator or halyard CLI tooling.
```

The Kustomize installation seems to work fine with the OSS Spinnaker container images in my limited testing: creating apps, pipelines and running pipelines, etc.